### ISSUE TITLE
Ipfs@0.30.1 could not be uploaded since yesterday, but can be online?

### ISSUE THREAD

**Issue by Despoinis** [ipfs/js-ipfs#1629]
Ipfs@0.31.0 or above can still be uploaded.
This is true for both Firefox and Chrome.

**Comment by alanshaw** [comment]
I don't understand the question, can you rephrase please?

**Comment by Despoinis** [comment]
> 
> 
> I don't understand the question, can you rephrase please?



> 
> 
> I don't understand the question, can you rephrase please?

It seems to be a network problem. After opening the VPN, I found that I can upload it.

**Comment by Despoinis** [comment]
> 
> 
> I don't understand the question, can you rephrase please?

It seems to be a network problem. After opening the VPN, I found that I can upload it.

**Comment by Despoinis** [comment]
> 
> 
> > I don't understand the question, can you rephrase please?
> 
> It seems to be a network problem. After opening the VPN, I found that I can upload it.

It can't be uploaded now with VPN, but it can still be uploaded above Ipfs@0.31.0. The network is too strange.

**Comment by alanshaw** [comment]
what are you uploading? do you mean downloading from the cdn?

**Comment by Despoinis** [comment]
> 
> 
> what are you uploading? do you mean downloading from the cdn?

I uploaded the image. At that time, I found that the version of Ipfs@0.30.1 could not be uploaded and downloaded from Japan and China. Now I have changed to the latest version and can upload and download. I didn’t use the latest version because
ReferenceError: AbortController is not defined

I solved it with a hack.
<script>
If (!AbortController) {
Var AbortController = function () { this.signal = { aborted: false, onabort: null } };
AbortController.prototype.abort = function () { };
}
</script>
